### ISSUE TITLE
[BUG] contenttype singular_name not used on button

### ISSUE THREAD

**Issue by jadwigo** [bolt/bolt#6882]
In the Bolt admin interface the "+ New X" button uses the contenttype slug, instead of the singular name.

To reproduce, a contentypes.yml with the following contenttype pages will result in the button in the screenshot:
```yml
pages:
  name: Pages
  singular_name: Page
  fields:
    [..]
```
in http://example.com/bolt/overview/pages

The expected result would be a button with 
![image](https://user-images.githubusercontent.com/1488214/29032026-5879ca0a-7b90-11e7-87a2-3e27faad39d8.png)

The actual result is:
![image](https://user-images.githubusercontent.com/1488214/29031844-d11ab57e-7b8f-11e7-89bf-b60654dc9e7d.png)

Details
-------

 - Relevant Bolt Version: 3.3 | master
 - Install type: Zip/tarball install 
 - PHP version: *
 - Used web server: *
 - For UX/UI issues: *

**Comment by GawainLynch** [comment]
Should be a just Twig use of "slug" instead of "name"/"single_name".

**Comment by SahAssar** [comment]
Wasn't this just fixed in #6876?

**Comment by GawainLynch** [comment]
Haha! I thought that, then was all "yeah, close but no banana!" 

Good call, @SahAssar :+1:

**Comment by jadwigo** [comment]
It appears so ... unfortunately my search for issues with the keyword "singular" did not return PR's with the same keyword.

Anyway.. fixed in 3.4 apparently

**Comment by GawainLynch** [comment]
> Anyway.. fixed in 3.4 apparently

3.2, 3.3, 3.4, 3.x & master … we work fast around here! :wink:
